### ISSUE TITLE
Tables: Ensure data table headers are properly identified

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#2533]
**OCIO Violation:** Ensure headers and cells are properly associated

**OCIO Notes:** This TABLE does not contain one or more: th elements (without an ARIA-assigned role); th elements with a role set to: rowheader | columnheader; or td elements with a role set to: rowheader | columnheader

**OCIO priority:** 5.5

More info in [this sheet](https://doimspp.sharepoint.com/:x:/s/NRRD/Ea-CVbPsGDpDkDLSIe6kZS8BNeTJ7C8_M7yRS86t6VEU2g?e=IXABwa) or the [ocio/amp dashboard](https://departmentofinterior.levelaccess.us/login.php)

**Instances**

- [ ] https://revenuedata.doi.gov/query-data/?dataType=Federal%20revenue%20by%20company

**Comment by eelzi-ONRR** [comment]
2024 Report:

Instances: 14
Occurrence: 4%
Severity: 10
Priority: 6.9

**Comment by lpgoldstein** [comment]
No longer relevant
